src/buttons/label.ts

```diff
@@ -0,0 +1,30 @@
+import {Directive, Renderer2, ElementRef} from '@angular/core';
+
+@Directive({selector: '[ngbButtonLabel]', host: {'[class.btn]': 'true'}})
+export class NgbButtonLabel {
+  constructor(private _renderer: Renderer2, private _elRef: ElementRef) {}
```

Comment:
Yep, will send a follow up PR to refactor it. Just didn't want to make too many changes in one PR.